docs/helm/configure-namespaced-helm-repos.md

```diff
@@ -0,0 +1,85 @@
+# Configure Namespace Scoped Chart Repositories using namespaced CRDs
+
+While the cluster-scoped CRD `HelmChartRepository` for Helm repository provides ability for admins to add Helm repositories as custom resources, regular users/developers need an admin's permission in order to add Helm repositories of their choice.
+
+To avoid permission elevation, use namespace scoped CRD `ProjectHelmChartRepository`.
```

Comment:
Request: Please check and confirm whether this procedure is the same for both cluster-scoped and namespace-scoped Helm charts in the **Developer Catalog**: https://docs.openshift.com/container-platform/4.9/applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.html#filtering-helm-charts-by-certification-level_configuring-custom-helm-chart-repositories

Reply:
The filtering is the same for either CRD: https://github.com/openshift/console/pull/10885

However, the UX to add/delete the CRD from the console UI is under discussion: https://coreos.slack.com/archives/CRDPF7YS2/p1642625858013100

Reply:
I misunderstood the purpose of "Configuring CA certificates and TLS configurations" in the original discussion. In this feature, the setups for the certificates and TLS config remain the same as before because the setups are for the access to the remote repository endpoint, not for the access to our CRDs. Please let me know if we are aligned.